pages/library/_viewId.vue

```diff
@@ -29,7 +29,7 @@
       <filter-button
         v-if="isSortable"
         :collection-info="collectionInfo"
-        :disabled="loading || !items.length"
+        :disabled="loading"
```

Comment:
I don't think filter have to be re-enabled in that case, if we have nothing to filter ^^

Reply:
If we are in a library, and apply a range of filters, such that there are no items, we can't unfiltered to show the items again.

Reply:
Nice catch!

Reply:
@Camc314 I that case I think we should keep a track of the original (unfiltered array) and check for that. Right now with what you did is possible to use filters in a library that has no items.